srt.go

```diff
@@ -46,6 +46,10 @@ func ReadFromSRT(i io.Reader) (o *Subtitles, err error) {
 
 		// Line contains time boundaries
 		if strings.Contains(line, srtTimeBoundariesSeparator) {
+			if len(s.Lines) == 0 {
+				// Print out the wrong number of lines
+				fmt.Errorf("error in the (%v) line", lineNum)
```

Comment:
As far as I understand it, this line won't print anything. What you want instead is:

```go
err = fmt.Errorf("astisub: line %d: no lines", lineNum)
return
```

Otherwise code will still panic and you won't have any details of the error.